app/consumer.py

```diff
@@ -57,13 +79,21 @@ def on_message(self, unused_channel, basic_deliver, properties, body):
             logger.error("ResponseProcessor failed", exception=e, tx_id=processor.tx_id)
 
 
-def main():
+def main(args):
     logger.debug("Starting consumer")
-    consumer = Consumer()
+
+    consumer = Consumer(args)
     try:
         consumer.run()
     except KeyboardInterrupt:
         consumer.stop()
 
+
+def run():
+    p = argparse.ArgumentParser(description=__doc__)
```

Comment:
Probably don't need arg support at this point - these services are stripped back as far as possible for simplicity. If we need these bits in the future we'll add them back in.